ntirpc/rpc/clnt.h

```diff
@@ -515,10 +506,27 @@ clnt_dg_ncreate(const int fd, const struct netbuf *raddr,
  */
 extern CLIENT *clnt_raw_ncreate(rpcprog_t, rpcvers_t);
 
+/*
+ * Client request processing
+ */
 int clnt_req_xid_cmpf(const struct opr_rbtree_node *lhs,
 		      const struct opr_rbtree_node *rhs);
 
-struct clnt_req *clnt_req_alloc(CLIENT *, struct timeval);
+static inline void clnt_req_fill(struct clnt_req *cc, struct rpc_client *clnt,
```

Comment:
So, it seems that clnt_req_fill() and clnt_req_setup() are always called as a pair.  Is there a reason to have separate calls for this?